settings_test.py

```diff
@@ -48,7 +48,7 @@ def _polite_tmpdir():
 # is just too annoying for tests, so disable it.
 CACHE_COUNT_TIMEOUT = -1
 CELERY_ROUTES = {}
-CELERY_ALWAYS_EAGER = True
+CELERY_ALWAYS_EAGER = False
```

Comment:
I was trying to test celery locally and must have committed this on accident. I'll remove.